pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -59,10 +59,7 @@ func (c *EtcdMembersController) sync(ctx context.Context, syncCtx factory.SyncCo
 	return nil
 }
 
-func (c *EtcdMembersController) reportEtcdMembers(recorder events.Recorder) error {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
```

Comment:
Spent way too long to figure out if the parent context was already cancellable until I found it 😅 

https://github.com/openshift/cluster-etcd-operator/blob/17978bc6552e27738e3b75b4d2c5c18101fcdd05/vendor/github.com/openshift/library-go/pkg/controller/controllercmd/cmd.go#L84-L92